backend/pom.xml

```diff
@@ -63,6 +63,13 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+<!--		&lt;!&ndash; For Maven &ndash;&gt;-->
+<!--		<dependency>-->
+<!--			<groupId>org.springframework.boot</groupId>-->
+<!--			<artifactId>spring-boot-starter-test</artifactId>-->
+<!--			<scope>test</scope>-->
+<!--		</dependency>-->
+
 

```

Comment:
why are you doing a lot of comments without meaning (just explain why commit or remove it or metion to leave) ... it is not a good practice